### ISSUE TITLE
dataansvarlig: KLE vs. Data

### ISSUE THREAD

**Issue by JosefAssad** [JosefAssad/SeMaWi#224]
Klassen "Data" har et felt "dataansvarlig", og den har et felt for KLE emner.

KLE emnerne har tilknyttet dataansvarlig, og det gør at man nu har to steder hvor den dataansvarlige kan defineres for et datasæt: direkte på datasættet og på et KLE emne.

Det kan potentielt give modsigende metadata.

Vi kan lade det stå foreløbigt, i øjeblikket er det væsentlige nok at vi bare er opmærksomme på denne detalje. Jeg tror ikke det er så akut. Men det ville være smart at bruge denne case som grundlag for at overveje, hvordan det er vi forankrer dataansvar.

Jeg ivl også nævne, at hvis vi betragter KLE emnets dataansvarlige som den autoritative for et datasæt, så skal vi overveje om et datasæt skal kunne knyttes til mere end et KLE emne, da man dermed åbner for muligheden for mere end 1 dataansvarlig for et datasæt.
Cc @ClockEnd @baffioso

**Comment by ClockEnd** [comment]
@JosefAssad Good point. Det løser sig når Organisationskomponenten er komplet med KLE på personniveau, vil jeg tro. Og if Databeskyttelsesforordningen tror jeg KLE vil få en vis betydning mht hjemmelsspørgsmålet - kan være vi har en utilsigtigt spin off/benefit med wiki'en.
